### ISSUE TITLE
"Due to ... used as left operand" false-positive

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1290]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | U2.0.15 |
| Language level | PHP 7.2 |
| IDE | PS 2019.1.1 Preview - 191.6707.42 |

![image](https://user-images.githubusercontent.com/421851/56009872-c37c8900-5cb7-11e9-9895-a0a45fc04348.png)

```php
/** @return string|array|null */
public static function old($key = null, $default = null) { }
```

This method could returns `NULL` or then empty, for instance, which will cause issues with `?:`.

**Comment by funivan** [comment]
Same story:
```php
class Link {
 function getHref() : ?string;
}
$link = new Link();
echo $link->getHref() ?? ''; // << should not be triggered
```

**Comment by kalessil** [comment]
Fixed!
